### ISSUE TITLE
Explore doesn't create automagic dashboards

### ISSUE THREAD

**Issue by jamarzka** [metabase/metabase#7527]
Using the new explorer looks good but when I click on a table it redirects to the auto dashboard page for the selected table with this error:

Something's gone wrong
We've run into an error. You can try refreshing the page, or just go back.

The network request for this: /api/automagic-dashboards/table/5 looks like this:
```
{
    "name": null,
    "transient_name": null,
    "transient_filters": null,
    "description": null,
    "creator_id": 1,
    "parameters": [],
    "related": {
        "indepth": [],
        "tables": [{
            "url": "/auto/dashboard/table/67",
            "title": "Calendar",
            "description": null
        }, {
            "url": "/auto/dashboard/table/66",
            "title": "Job issue",
            "description": null
        }],
        "more": []
    }
}
```
I see this in the logs:
```
05-02 22:10:56 INFO automagic-dashboards.core :: Applying heuristic null to Log.
05-02 22:10:56 INFO automagic-dashboards.core :: Dimensions bindings:
null
05-02 22:10:56 INFO automagic-dashboards.core :: Using definitions:
Metrics:
null
Filters:
null
nil
05-02 22:10:56 INFO automagic-dashboards.populate :: Adding 0 cards to dashboard null:

05-02 22:10:56 DEBUG metabase.middleware :: GET /api/automagic-dashboards/table/77 200 (192 ms) (11 DB calls). Jetty threads: 8/50 (3 busy, 5 idle, 0 queued)
```
The explorer works for the sample database.

- Your browser and the version: Chrome 66.0.3359.139
- Your operating system: Ubuntu 18.04
- Your databases: Postgres
- Metabase version: 0.29
- Metabase hosting environment: Ubuntu 16.04
- Metabase internal database: Postgres

**Comment by jamarzka** [comment]
It's working for me now with the latest from master

**Comment by salsakran** [comment]
I believe this should be fixed via #7548 in the more general case.
